wallet-connect-test-bench/front-end/src/writing_to_blockchain.ts

```diff
@@ -100,7 +100,8 @@ export async function setValue(
     isPayable: boolean,
     dropDown: string,
     input: string,
-    cCDAmount: string
+    cCDAmount: string,
+    maxEnergy: string
 ) {
     let receiveName = `${CONTRACT_NAME}.set_u8_payable`;
```

Comment:
Maybe we could log all the inputs we are sending to the wallet here?

Like we do for the issuer frontends. That makes it simpler to debug.